module.json

```diff
@@ -19,10 +19,10 @@
         "pf1",
         "pf2e",
         "D35E",
-        "sw5e"
+        "sw5e",
+        "sfrpg"
     ],
-    "languages": [
-        {
+    "languages": [{
```

Comment:
The change on these two lines needs to be reverted (slight formatting inconsistency).
Same for line 66 and 88.